demo-project/src/demo_project/pipelines/data_ingestion/pipeline.py

```diff
@@ -27,21 +27,21 @@ def create_pipeline(**kwargs) -> Pipeline:
                 func=apply_types_to_companies,
                 inputs="companies",
                 outputs="int_typed_companies",
-                name='apply_types_to_companies',
-                tags='companies'
+                name="apply_types_to_companies",
+                tags="companies",
             ),
             node(
                 func=apply_types_to_shuttles,
                 inputs="shuttles",
                 outputs="int_typed_shuttles@pandas1",
-                name='apply_types_to_shuttles',
-                tags='shuttles'
+                name="apply_types_to_shuttles",
+                tags="shuttles",
```

Comment:
Another autofix from `ruff format` - makes sense, since now single and double quotes are mixes within the same `node` definition. I'm curious why previous formatter didn't flag that.

Reply:
The formatter was [run](https://github.com/kedro-org/kedro-viz/pull/2149/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52L30) only on `package/` . The `demo-project` is outside of the formatter.